src/Microsoft.AspNetCore.OData/ODataController.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT License.  See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNet.OData.Common;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.OData.Edm;
+using Microsoft.OData.Edm.Csdl;
+
+namespace Microsoft.AspNet.OData
+{
+    /// <summary>
+    /// Defines a base class for OData controllers that support writing and reading data using the OData formats.
+    /// </summary>
+    //[ODataFormatting]
```

Comment:
Add todo to address later?

Reply:
Yes, as that class is added. Would you rather see if removed in this commit?

Reply:
I'll leave it up to you as long as it's eventually removed. Just wanted to make sure you were aware of it.